public/static/locales/eng/the-database.json

```diff
@@ -7,7 +7,11 @@
         "title": "The Samrómur database",
         "beginning": "Samrómur is an open and accessible database of voices that everyone is free to use when developing software in Icelandic.",
         "middle": "The database consists of sentences and audio clips from the reading of those sentences as well as metadata. Each entry in the database contains FLAC audio clips and the corresponding text file.",
-        "end": "The first release of 100,000 utterances has been released on <2>OpenSLR</2>. Subscribe to the newsletter below to be notified when further datasets becomes accessible."
+        "preface-releases": "There are currently three releases of data sets from Samrómur.",
```

Comment:
on line ten you call them data sets but on line 14 you call them datasets.

Reply:
going for data sets in both